views/Main.react.js

```diff
@@ -574,8 +574,20 @@ class MainPage extends Component {
     default_acl = options;
     this.starAcl(options, this.state.star)
     this.setState({
-      totalStudies: acl.length,
-      totalSubjects: options.length,
+      // study determination: at least one upper case
+      totalStudies: acl.filter(s=> /[A-Z]/.test(s.study)).length,
```

Comment:
What do you think about using this filter here? 

https://github.com/AMP-SCZ/dpdash/blob/94d6b275bb73253fe6d063b89d239804fb82a992/server/utils/passport/local-login.js#L84

Maybe I need more context, so we want to make sure that these sites are not part of the list?

Reply:
Sorry, i'm unclear as to what the goal is here and I want to make sure we get this right.

Reply:
Not sure how your two comments are related to the line in question.

Reply:
I think the comment is that we could filter the `acl` on the server side so that this line would remain unchanged. It is confusing to me why `totalStudies` wouldn't be the total studies but instead a filtered list of studies.

Reply:
this is React logic. I think the idea is to have the server provide this filtered data to the client.